bridge/slack/slack.go

```diff
@@ -288,6 +288,12 @@ func (b *Bslack) sendRTM(msg config.Message) (string, error) {
 		return "", err
 	}
 
+	// Handle prefix hint for unthreaded messages.
+	if msg.ParentID == "unthreaded" {
+		msg.ParentID = ""
+		msg.Text = fmt.Sprintf("thread reply: %s", msg.Text)
```

Comment:
@42wim I moved this into the slack bridge as per your request, but in retrospect, why is it that we're not notifying other protocols of unthreading?

Reply:
Other protocols also get notified ? As the notification is inside the `msg.ParentID`?